src/app.py

```diff
@@ -38,19 +38,28 @@
 answer_logger = Logger()
 
 
-def getUserName():
-    return current_user.username
-
-def getUserId():
-    return current_user.id
+def getUserName():
```

Comment:
You shouldn't have to make these changes. I expect all your changes to be stepper related/